go.mod

```diff
@@ -1,17 +1,17 @@
 module github.com/prometheus/common
 
 require (
-	github.com/go-kit/log v0.1.0
+	github.com/go-kit/log v0.2.0
 	github.com/golang/protobuf v1.4.3
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/matttproud/golang_protobuf_extensions v1.0.1
 	github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f
 	github.com/prometheus/client_golang v1.11.0
```

Comment:
Can you update this one as well to v1.12.1 or later?

Reply:
Done